src/test/run-pass/specialization/defaultimpl/specialization-trait-item-not-implemented.rs

```diff
@@ -8,30 +8,27 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-// Test that non-method associated functions can be specialized
-
 #![feature(specialization)]
 
```

Comment:
Comment:

---

Tests that we can combine a default impl that supplies one method with a full impl that supplies the other, and they can invoke one another.